articles/macos-setup-experience.md

```diff
@@ -178,6 +178,8 @@ If you configure software and/or a script for setup experience, users will see a
 
 This window shows the status of the software installations as well as the script exectution. Once all steps have completed, the window can be closed and Setup Assistant will proceed as usual.
 
+> Currently, software set to [automatic install](https://fleetdm.com/guides/automatic-software-install-in-fleet) is also installed during Setup Assistant but won't appear in the window. All software, automatic and setup experience, isn't installed in any particular order. The script always runs after setup experience software is installed.
```

Comment:
```suggestion
> The setup experience script always runs after setup experience software is installed. Currently, software that [automatically installs](https://fleetdm.com/guides/automatic-software-install-in-fleet) and scripts that [automatically run](https://fleetdm.com/guides/policy-automation-run-script) are also installed and run during Setup Assistant but won't appear in the window. Automatic software and scripts may run before or after setup the experience software and script. They aren't installed/run in any particular order.
```

Reply:
@jahzielv sounds good 👍 

Heads up that I made a few tweaks to the docs in my latest commit.

Is there a good spot on the Setup experience page in the UI so IT admins are aware?